Dockerfile

```diff
@@ -1,16 +1,46 @@
-FROM 		phusion/baseimage
-MAINTAINER 	Casey Bisson <casey.bisson@gmail.com
+FROM alpine:3.2
+MAINTAINER 	Casey Bisson <casey.bisson@gmail.com>
 
-# Update Apt
-RUN apt-get update
+# Alpine packages
+RUN apk --update \
+    add \
+        jq \
+        curl \
+        bash \
+        ca-certificates && \
+    curl -Ls https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk > /tmp/glibc-2.21-r2.apk && \
```

Comment:
It occurs to me that a musl build for something running DNS is something we'd want to test out quite a bit. One of the big behavioral differences between musl and libc is the DNS client library. Not sure if that'll turn out to be relevant for a Go program, but worth checking.